### PR TITLE
Updated R4 CarePlan Resource

### PR DESCRIPTION
**Pull request by kckeim**
Formatting updates to be consistent across parameter notes, including punctuation, and matching _revinclude to other parameters' styling.
Verified good links throughout document and have validated that example API calls behave as noted with examples.

Description
----

- Miscellaneous formatting changes (e.g. adding periods/punctuation) for consistency across parameters
- Parameters table - Required flag for _revinclude changed to match others'
- Parameter notes - _id changed wording to note interaction with _revinclude
- Parameter notes - _revinclude modified structure to largely match other parameters' styling

PR Checklist
----

- Parameters table before _revinclude change:

image

After:
image

*****************************

- _id parameter details before change:
image

After:
image

*****************************

-  _revinclude section of parameter notes before change:
image

After:
image

### CODE REVIEW

File: content/millennium/r4/clinical/care-provision/care-plan.md [hunk 1]
```diff
@@ -60,19 +60,19 @@ _Implementation Notes_
...
+  - May not be provided with any other parameters, except for `_revinclude` as indicated below.
   - When provided, `_count` is ignored.
 
 - The `date` parameter
-  - When provided, must use both `ge` and `le` prefixes in the same search
-    - The lower value must have the `ge` prefix and the higher value must have the `le` prefix
-    - If date precision must be consistent
+  - When provided, must use both `ge` and `le` prefixes in the same search.
+    - The lower value must have the `ge` prefix and the higher value must have the `le` prefix.
+    - If date precision must be consistent.
```
Comment (amcginn) [inline content/millennium/r4/clinical/care-provision/care-plan.md:75]: is "If" intentional?

File: content/millennium/r4/clinical/care-provision/care-plan.md [hunk 2]
```diff
@@ -83,9 +83,9 @@ Notes:
   - Only supports the code `assess-plan`.
   - This can be combined with the patient parameter.
 
-- The `_revinclude` parameter may be provided once with the value `Provenance:target`. Example: `_revinclude=Provenance:target`
-
-- The `_revinclude` parameter may be provided with the `_id/patient` parameter. Example: `_id=178866310&_revinclude=Provenance:target`
+- The `_revinclude` parameter 
+  - May be provided once with the value `Provenance:target`. Example: `_revinclude=Provenance:target`
+  - May be provided with the `_id/patient` parameter. Example: `_id=178866310&_revinclude=Provenance:target`
```
Comment (amcginn) [inline content/millennium/r4/clinical/care-provision/care-plan.md:88]: let's split _id and patient so it doesn't look like a single parameter

**Review by amcginn — CHANGES_REQUESTED** [review]

File: content/millennium/r4/clinical/care-provision/care-plan.md (hunk 2 above)
Comment (kckeim) [reply content/millennium/r4/clinical/care-provision/care-plan.md:88]: Adjusted in latest commit to the branch
image

File: content/millennium/r4/clinical/care-provision/care-plan.md (hunk 1 above)
Comment (kckeim) [reply content/millennium/r4/clinical/care-provision/care-plan.md:75]: No.
Adjusted in latest commit to the branch
image

**Review by RicardoCrespo-ora — APPROVED** [review]

**Review by amcginn — APPROVED** [review]

**Pull request merged**